core/sup_tree_core/gear_config_poller.ex

```diff
@@ -29,6 +29,13 @@ defmodule AntikytheraCore.GearConfigPoller do
     {:ok, %{last_checked_at: 0}, @interval}
   end
 
+  @impl true
+  def handle_cast(:reload, state) do
+    checked_at = System.system_time(:seconds)
+    GearConfig.load_all(state[:last_checked_at])
+    {:noreply, %{state | last_checked_at: checked_at}, @interval}
```

Comment:
Don't copy existing code.
Let's make a private function to extract the common part.

Reply:
fixed in 75f01f2